### PR TITLE
Form_Page-for_Portfolio

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Added form__page section with form tags.
-Added styling for all the tags in the section with flexBox.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -708,3 +708,126 @@ body {
...
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  align-self: flex-start;
+}
+
+.input__btn:hover {
+  border: 1px solid #396df2;
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+}
```
Comment (safafa) [inline style.css]: - [ ] Kindly add the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) style as shown in the figma template.
image

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @addisalemseifu and @karelvanoordt , You did a great job so far 👏🏼 👏🏼 

- the form is working as expected.
- linter checks are passing.

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (ZeenatLawal) [reply style.css]: - [ ] The issue here has not been properly resolved. The button's background colour doesn't change on hover, kindly fix that.

FireShot Capture 105 - Addisalem seifu Portofolio - raw githack com

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @addisalemseifu and @karelvanoordt, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] _Kindly check the comments above, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewer, and please check to see that the changes are working as intended._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addisalemseifu** [comment]
The hover effect is working on my machine. I can not understand why it is not working here.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @addisalemseifu, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**